test/Ocelot.UnitTests/Configuration/SecurityOptionsCreatorTests.cs

```diff
@@ -38,7 +38,7 @@ public void should_create_security_config()
                 }
             };
 
-            var expected = new SecurityOptions(ipAllowedList, ipBlockedList);
+            var expected = new SecurityOptions(ipAllowedList, ipBlockedList, false);

```

Comment:
You need NOT to specify third parameter if you have defined 2 constructors! That's how to achieve the backward compatibility.

Reply:
Done!